packages/notte-sdk/src/notte_sdk/endpoints/workflows.py

```
@@ -508,7 +508,10 @@ def run(
                         decoded = WorkflowsClient.decode_message(log_msg)
 
                         if message["type"] == "log":
-                            logger.opt(colors=True).info(decoded)
+                            try:
+                                logger.opt(colors=True).info(decoded)
+                            except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)